web/src/routes/results/[eid]/+page.svelte

```diff
@@ -0,0 +1,259 @@
+<script lang="ts">
+	import Button from "$lib/button.svelte";
+	import Dialog from "$lib/dialog.svelte";
+	import Panel from "$lib/panel.svelte";
+
+	import {
+		type ElectionOutcome,
+		type ElectionOutcomeResult,
+		elections,
+		error,
+		fetching,
+		user,
+	} from "../../../store";
+	import { goto } from "$app/navigation";
+	import { API } from "$lib/endpoints";
+	import { _getCurrentElection, _getElections } from "../../+layout";
+
+	export let data: { data: ElectionOutcome };
+
+	$: electionOutcome = data.data;
+
+	$: election = electionOutcome.election;
+	$: resultsByRound = electionOutcome.results.reduce((res, cur) => {
+		return {
+			...res,
+			[cur.round]: res[cur.round]
+				? [...res[cur.round], cur].sort((a, b) =>
+						a.voteCount != b.voteCount ? b.voteCount - a.voteCount : b.isRejected ? -1 : 1,
+				  )
+				: [cur],
+		};
+	}, {} as { [key: string]: Array<ElectionOutcomeResult> });
+	$: winner = electionOutcome.results
+		.filter((e) => e.isElected)
+		.map((e) => e.name)
+		.join(", ");
+	$: date = new Date(electionOutcome.date).toUTCString();

```

Comment:
Would be nicer to use DateTimeFormat, something like so:
```
$: date = new Intl.DateTimeFormat("en-GB", {
		dateStyle: "full",
		timeStyle: "long",
	}).format(new Date(electionOutcome.date));
```

This would format the date as `Sunday 28 July 2024 at 20:32:30 BST`, which reads better with the "at" separating the date and time IMO.